javascript/javascript2/week1/exercise-1/cardio.js

```diff
@@ -0,0 +1,14 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function findShortestWord(danishWords) {
+  let shortestWord = "";
+  for (let i = 0; i < danishWords.length; i++) {
+    if (shortestWord === "") {
+      shortestWord = danishWords[i];
+      console.log(shortestWord.length);
```

Comment:
this `if` clause is not really necessary, why not setting `shortestWord = danishWords[0]` in line 3 ? Can you improve your code with that change and see how it simplifies?

Reply:
Got it, Thnak you